index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
+        <div class="tonic-heading">Tonic</div>
+        <div class="canopy-container">
+            <div class="canopy-header">CANOPY</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">Back End Dev</div>
+            <div class="canopy-img"><img src="images/counter.png" alt="counter"></div>
+            <div class="canopy-text">2015</div>
```

Comment:
We use `<div>`s when you need to divide (that's what divs are supposed to be used for) sections of your page from one another. Header/footer from content for instance, or individual posts on a blog. However, you shouldn't use them when semantically another element would make sense.

- [ ] You can use `<ul>` and `<li>` in this section. :+1: